src/Core/Scheduler/Providers/TMS_Provider.js

```diff
@@ -7,12 +7,17 @@ function TMS_Provider() {
 
 TMS_Provider.prototype.preprocessDataLayer = function preprocessDataLayer(layer) {
     if (!layer.extent) {
-        throw new Error(`Missing extent property for layer '${layer.id}'`);
+        // default to the full 3857 extent
+        layer.extent = new Extent('EPSG:3857',
+            -20037508.342789244, 20037508.342789244,
```

Comment:
How did you choose those bounds ? Here are [other ones](http://epsg.io/3857), and I can find other too for this projection.

Reply:
From proj4:
```
itowns.proj4('EPSG:3857').forward([-180, 0])
(2) [-20037508.342789244, -7.081154551613622e-10]
```
y-min/y-max are set to have a square extent (which seems to be correct).
Note that this is only a default extent value: you're free to override it.

Reply:
Ok for letting it as it is